doc/src/modules/index.rst

```diff
@@ -29,24 +32,19 @@ access any SymPy module, or use the list below:
    core.rst
    crypto.rst
    diffgeom.rst
-   solvers/diophantine.rst
    discrete.rst
    evalf.rst
    functions/index.rst
    geometry/index.rst
    holonomic/index.rst
-   solvers/inequalities.rst
-   integrals/integrals.rst
-   integrals/g-functions.rst
+   integrals/index.rst
    interactive.rst
    liealgebras/index.rst
    logic.rst
    matrices/index.rst
    ntheory.rst
    numeric-computation.rst
-   solvers/ode.rst
```

Comment:
I'm -1 to these changes. Having these on the front page makes them easier to find.

Reply:
Unless the maxdepth makes them show up under solvers? Let me build the docs here and see what it looks like.

Reply:
No, it looks like it doesn't.

Reply:
I found that other documents such as `tensor`, `series`, `integrals` etc have same problem. I fixed them so that subdocs are exposed here by maxdepth. Please build again and check it!

Reply:
The removed sections still don't appear on the main index page.

Reply:
@asmeurer 
I increased the main index page toctree depth from 2 to 3. This will make subsections of other documents easier to find.